keylime/types.py

```diff
@@ -1,9 +1,48 @@
+import sys
 import typing
 
+from keylime.ima.ima_dm import Policies
```

Comment:
The inclusion of this here could cause some issues down the road. Maybe the types would have to be moved into this file here or into keylime.ima.ima_dm_types.

Reply:
Maybe we can put all the IMA related types into `keylime.ima.types` and keep only the more general ones in `keylime.types`?

Reply:
I moved it now.